app/services/identity_api.rb

```diff
@@ -5,15 +5,16 @@ class ApiError < StandardError
 
   FIVE_SECONDS = 5
 
-  def self.get_teacher(_uuid)
-    # Stub this method for now while the required API endpoint is being built
-    {
-      userId: "29e9e624-073e-41f5-b1b3-8164ce3a5233",
-      email: "kevin.e@digital.education.gov.uk",
-      firstName: "Kevin",
-      lastName: "E",
-      dateOfBirth: "1990-01-01",
-    }
+  def self.get_teacher(uuid)
+    if FeatureFlag.active?(:identity_api_always_timeout)
+      raise Faraday::TimeoutError, "Time-out feature flag enabled"
+    end
+
+    response = new.client.get("/api/v1/teachers/#{uuid}")
+
+    raise ApiError, response.reason_phrase unless response.status == 200
+
+    response.body
```

Comment:
@malcolmbaig we could use the result to populate the new User class in the same way as the `get_users` method? This way we have a mapping to the api response attributes in one place and do not need to reference them in the views

Reply:
Thanks @rmdasilvaD2l , updated 👍🏼